pkg/sentry/seccheck/sinks/remote/remote.go

```diff
@@ -74,14 +91,20 @@ func setupSink(config map[string]any) (*os.File, error) {
 	if !ok {
 		return nil, fmt.Errorf("endpoint %q is not a string", addrOpaque)
 	}
-	return setup(addr)
+	file, _, err := setupWithVersion(addr)
```

Comment:
See comment below, if `setupWithVersion()` handles the remote's version, `setupWithVersion()` doesn't need to return the version (which is ignored here anyways).